src/runtime/flow_node_handler/activity_handler/call_activity_handler.ts

```diff
@@ -35,6 +35,11 @@ export class CallActivityHandler extends ActivityHandler<Model.Activities.CallAc
   private processModelUseCases: IProcessModelUseCases;
   private resumeProcessService: IResumeProcessService;
 
+  private subProcessInstanceId: string;
+  private terminatedSubscription: Subscription;
```

Comment:
Die Namen bitte ausschreiben, damit hier jeder verstehen kann, auf welches Terminated Event hier gewartet wird.

```suggestion
  private subProcessTerminatedSubscription: Subscription;
```